daemon/algod/api/spec/v1/model.go

```diff
@@ -524,6 +524,11 @@ type Transaction struct {
 	// required: true
 	ApplicationCall *ApplicationCallTransactionType `json:"app,omitempty"`
 
+	// CompactCert
+	//
+	// required: true
```

Comment:
why is required true?

Reply:
I actually have no idea.  I don't know what's going on with the `required:` annotations there for swagger (I think).  If you can tell me what it should be instead, I'm happy to change it.  I just copied it from other entries that seemed semantically identical (like the `ApplicationCall` above it).